client/galaxy/scripts/components/Workflow/Editor/workflowRecommendations.js

```diff
@@ -0,0 +1,74 @@
+import $ from "jquery";
```

Comment:
Everything is already in Vue, I am not why this helper file is necessary. Also jquery should not be used in new code unless absolutely necessary.

Reply:
I needed to communicate `Node.vue` and `WorkflowRecommendationsVue`. I will look out for a way to avoid it and jquery too.

Reply:
I have removed the helper file, but not sure how to avoid the mounting. The new template should be appended to the body tag.

Reply:
Also, I have found one small issue with the height of the D3 tree. When the number of items becomes > 15, it gets distorted. Working on it.

Reply:
![z203](https://user-images.githubusercontent.com/3022518/77862689-6443bb80-721d-11ea-9e73-371350951e94.png)
![z202](https://user-images.githubusercontent.com/3022518/77862690-6574e880-721d-11ea-8529-d859703391d6.png)
![z201](https://user-images.githubusercontent.com/3022518/77862691-660d7f00-721d-11ea-930d-1e3bdf28da98.png)